myList.rb

```diff
@@ -0,0 +1,20 @@
+# Create a class MyList that has an instance variable @list
+require_relative 'MyEnumerable'
+class MyList
+include MyEnumerable
+    def initialize(items)
```

Comment:
### Optional
- [ ] It would be nice if we fix the error in the number of arguments as in the screenshot below. The code run as expected
![Screenshot from 2022-07-21 17-52-53](https://user-images.githubusercontent.com/46388113/180281224-0240291c-107f-45a9-9b57-1678718e66ab.png)

Reply:
Fixed